src/c_api/c_api_symbolic.cc

```diff
@@ -87,7 +89,33 @@ int MXSymbolCreateAtomicSymbol(AtomicSymbolCreator creator,
                                const char **keys,
                                const char **vals,
                                SymbolHandle *out) {
-  return NNSymbolCreateAtomicSymbol(creator, num_param, keys, vals, out);
+  nnvm::Symbol *s = new nnvm::Symbol();
+  API_BEGIN();
+  const nnvm::Op* op = static_cast<const nnvm::Op*>(creator);
+  std::unordered_map<std::string, std::string> kwargs;
+  for (nn_uint i = 0; i < num_param; ++i) {
+    bool flag = false;
+    for (const auto &k : kHiddenKeys) {
```

Comment:
I am fine merging it if it will get eventually deleted after evolution.

For speed testing, simply do elementwise add composition and measure ops /sec